contentful/content-management/changes/20241202-1200-find-and-delete-missing-references.js

```diff
@@ -99,22 +97,23 @@ async function processField({
     locale,
     fieldKey,
     fieldValue,
-} = {}) {
+}) {
     if (Array.isArray(fieldValue)) {
         await removeMissingReferencesInArray({
-            client: client,
-            entry: entry,
-            validEntryIds: validEntryIds,
-            locale: locale,
-            fieldKey: fieldKey,
+            client,
+            entry,
+            validEntryIds,
+            locale,
+            fieldKey,
             fieldItems: fieldValue,
         });
-    } else if (fieldKey.sys?.type === "Link") {
+    } else if (fieldValue.sys?.type === "Link") {
```

Comment:
I'm not sure what went wrong with this when it got pushed, feel like this isn't your fault but 
```suggestion
    } else if (fieldValue?.sys?.type === "Link") {
```

Reply:
Good spot. Fixed